### PR TITLE
correct linting issues from ESLint

### PR DESCRIPTION
**Pull request by expe**
DO NOT MERGE until OBT is final.

Changing code to be compliant with latest OBT.

Note: There are still some warnings regarding missing descriptions in JSDoc. @wheresrhys You're probably more suitable for fixing the warnings.

### CODE REVIEW

File: cookie.js [hunk 1]
```diff
@@ -1,6 +1,3 @@
-var pluses = /\+/g;
```
Comment (expe) [inline cookie.js]: plusses isn't used

**Comment by wheresrhys** [comment]
Most of the code here I copied across from some old FT.com code, so I'm not particularly well versed in why the code is the way it is

**Comment by expe** [comment]
@AlbertoElias reckons o-cookie is unmaintained / abandoned and these changes probably aren't necessary. @triblondon do you have any thoughts?

**Comment by expe** [comment]
o-cookie is deprecated, closing this PR.

**Pull request closed without merging**